db/migrate/20231129011739_create_users.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+      t.string :name, null: false
+      t.string :photo, null: false
+      t.text :bio
+
+      # Add an index for the photo column
+      t.index :photo, unique: true
+
+      t.timestamps
+    end
```

Comment:
- [ ] In your migrations, the method `PostsCounter` should be present as seen in the EDR design given 

### Design:
![image](https://github.com/sagieramos/blogpost/assets/50990925/cca15f39-7b80-44e7-9ebe-3cb4d74def8b)